content/en/security/cloud_security_management/guide/public-accessibility-logic.md

```diff
@@ -202,6 +202,16 @@ An [SQS Queue][32] (`aws_sqs_queue`) is considered publicly accessible if:
 
 See [Amazon SQS security best practices][33] for more information about public SQS queues.
 
+### Amazon Lambda Function
+
+A [Lambda Function][58] (`aws_lambda_function`) is considered publicly accessible if:
+
+| **Criteria** | **Explanation** |
+|--------------|-----------------|
+|The function has a policy that allows any principal (`principal_policy` or `principal_aws`) set to `"*"`. |This setting makes the function accessible to everyone in the world or to any authenticated AWS user.|
+
+See [Best practices for working with AWS Lambda Function][59] for more information about public Lambda functions.
```

Comment:
```suggestion
See [Best practices for working with AWS Lambda function][59] for more information about public Lambda functions.
```

Reply:
Apologies for the back and forth here, but just requesting this minor update for the reasons outlined [here](https://github.com/DataDog/documentation/pull/22419#discussion_r1558047907).

Additionally, if you open the link, you'll find that `function` is not capitalized at the destination, either.